pyflakes/checker.py

```diff
@@ -1095,8 +1095,16 @@ def TRY(self, node):
     TRYEXCEPT = TRY
 
     def EXCEPTHANDLER(self, node):
-        # 3.x: in addition to handling children, we must handle the name of
-        # the exception, which is not a Name node, but a simple string.
-        if isinstance(node.name, str):
-            self.handleNodeStore(node)
+        if not isinstance(node.name, str):
```

Comment:
please use `if PY2:` instead of type inspection, so that it is more clear that this is version specific code